frontend/src/routes/_dashboard/index.lazy.tsx

```diff
@@ -276,24 +341,30 @@ export function ProjectList() {
 
           {!noResults && (
             <div
-              className={cn(
-                "my-4",
-                slug !== debouncedValue && "opacity-40 pointer-events-none"
-              )}
+              className={cn("my-4 block", {
+                hidden: noArchivedProjects,
+                "opacity-40 pointer-events-none": slug !== debouncedValue
+              })}
             >
               <Pagination
                 totalPages={totalPages}
                 currentPage={page}
                 perPage={per_page}
                 onChangePage={(newPage) => {
                   navigate({
-                    search: { slug, page: newPage, per_page, sort_by },
+                    search: { slug, page: newPage, per_page, sort_by, status },
                     replace: true
                   });
                 }}
                 onChangePerPage={(newPerPage) => {
                   navigate({
-                    search: { slug, page: 1, per_page: newPerPage, sort_by },
+                    search: {
+                      slug,
+                      page: 1,
+                      per_page: newPerPage,
+                      sort_by,
+                      status
+                    },
```

Comment:
You can simplify this with : 
```suggestion
                    search: {
                      ...filters,
                      page: 1,
                      per_page: newPerPage,
                    },
```